loop/routes/rooms.js

```diff
@@ -187,10 +191,11 @@ module.exports = function (apiRouter, conf, logError, storage, auth,
    * Create a new room with the given information
    **/
   apiRouter.post('/rooms', auth.requireHawkSession,
-    validators.requireParams('roomName', 'roomOwner', 'maxSize'),
+    validators.requireParams(['context', 'roomName'], 'roomOwner', 'maxSize'),
```

Comment:
Is it actually required? In case it's not specified, I believe it should continue to work.